go/tasks/plugins/hive/config/config.go

```diff
@@ -25,31 +25,34 @@ func MustParse(s string) config.URL {
 	return config.URL{URL: *r}
 }
 
+type ClusterConfig struct {
+	Label string `json:"label" pflag:",Label of the service cluster"`
+	Limit int    `json:"limit" pflag:",Resource quota of the service cluster"`
```

Comment:
can you add what the units are either in the pflag or variable name (e.g. LimitInFoo)